src/auth/settings.js

```diff
@@ -0,0 +1,7 @@
+export const settings = {
+    authority: process.env.REACT_APP_AUTHORITY,
+    client_id: process.env.REACT_APP_CLIENT_ID,
+    redirect_uri: process.env.REACT_APP_REDIRECT_URL,
+    response_type: process.env.REACT_APP_RESPONSE_TYPE,
+    scope: process.env.REACT_APP_SCOPE
+}
```

Comment:
Please remove authentication data from the URL after successful authentication with something like this
```
onSigninCallback: () => {
    window.history.replaceState(
        {},
        document.title,
        window.location.pathname
    )
}
```

Reply:
Okay, I'll know how to do it right.